Utilities/bootstrap

```diff
@@ -915,6 +915,13 @@ def main():
     sandbox_path = os.path.join(build_path, ".bootstrap")
     target_path = os.path.join(build_path, build_target)
 
+    # Confirm rsync is available.
+    try:
+        FNULL = open(os.devnull, 'w')
+        subprocess.check_call(['rsync', '--version'], stdout=FNULL, stderr=subprocess.STDOUT)
```

Comment:
I think you might be able to just use on of the other methods like check_output to simplify this and avoid needing to open a file yourself

Reply:
Thanks for the feedback :) 

I'm not sure if I'm mis-reading the documentation, but it seems to me you would still need to catch the error; otherwise, you would get a full stack track when it exists with a non-zero code: 

https://docs.python.org/2/library/subprocess.html#subprocess.check_output

Reply:
Yeah, though I am not sure if that redirects stderr.

Reply:
@freak4pc can you check the docs?

Reply:
@freak4pc Daniel meant using check_output so we can avoid using open():

```
subprocess.check_output(['rsync', '--version'], stderr=subprocess.STDOUT)
```

Reply:
Oh I see. Should I make a quick PR for this or do you want to just go ahead?

Reply:
Would be awesome if you can make a PR to update this!